packages/did-auth-siop-op-authenticator/__tests__/shared/mockedData.ts

```diff
@@ -162,7 +175,7 @@ export const vpSingle = [
 export const vpMultiple = [
   {
     format: 'ldp_vp',
-    location: 'id_token',
+    location: 'presentation_definition',
```

Comment:
This doesn't make sense. We are in the VP here. You cannot place a VP in a definition

Reply:
fixed